src/pkg/retention/policy/rule/latestk/evaluator_test.go

```diff
@@ -0,0 +1,57 @@
+package latestk
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/goharbor/harbor/src/pkg/retention/policy/rule"
+	"github.com/goharbor/harbor/src/pkg/retention/res"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEvaluator_New(t *testing.T) {
```

Comment:
Can we use the `github.com/stretchr/testify` testing framework to drive the test cases as I did in the `or/processor_test.go`?